### ISSUE TITLE
(Linux) Applications in Split Tunnel are blocked by firewall

### ISSUE THREAD

**Issue by stenya** [ivpn/desktop-app#189]
# Bug report

## Describe your environment

* OS name and version: Linux (tested on Ubuntu)
* IVPN app version: v3.8.20, v3.9.1 (NO issue on v3.8.7)

### Steps to reproduce:

1. Connect VPN
2. Ensure Firewall is enabled
3. Start Firefox in Split Tunnel **=> Firefox has no connectivity** 
4. Disable Firewall **=> Firefox has connectivity and correctly working in the Split Tunnel environment**

**Comment by stenya** [comment]
The issue happens because aggressive block rules which were added in [commit b2533df](https://github.com/ivpn/desktop-app/commit/b2533dfbae0041c8898f2d4dd98a71d366bbf4f2) (related to ticket https://github.com/ivpn/desktop-app/issues/93)

Line 156:
```
      # Aggressive block!
      # Note! If the packet does not match any IVPN rule - DROP it.
      # It prevents traversing packet analysis to the rest rules (if defined) and avoids any leaks
      # This will block all user-defined firewall rules!
      ${IPv6BIN} -w ${LOCKWAITTIME} -A ${OUT_IVPN} -j DROP
      ${IPv6BIN} -w ${LOCKWAITTIME} -A ${IN_IVPN}  -j DROP
```

Line 233:
```
    # Aggressive block!
    # Note! If the packet does not match any IVPN rule - DROP it.
    # It prevents traversing packet analysis to the rest rules (if defined) and avoids any leaks
    # This will block all user-defined firewall rules!
    ${IPv4BIN} -w ${LOCKWAITTIME} -A ${OUT_IVPN} -j DROP
    ${IPv4BIN} -w ${LOCKWAITTIME} -A ${IN_IVPN}  -j DROP
```

**Comment by stenya** [comment]
v3.9.3
@gorkapernas it is necessary to double-check https://github.com/ivpn/desktop-app/issues/93
Let me know if you do not have the possibility to do it.

**Comment by gorkapernas** [comment]
@stenya I confirmed that the issue is fixed on v3.9.3, tested on Ubuntu 20.04 and Fedora 34, but I'll spend some time testing https://github.com/ivpn/desktop-app/issues/93 as you mention and let you know if everything is okay.

**Comment by gorkapernas** [comment]
@stenya tested https://github.com/ivpn/desktop-app/issues/93 on Ubuntu 20.04 and Fedora 35, everything works as expected, there are no IPv6 leaks.

**Comment by stenya** [comment]
Released in v3.9.8
